yarn.lock

```diff
@@ -16826,9 +16826,9 @@ __metadata:
   languageName: node
   linkType: hard
 
-"vite@npm:^5.4.15":
-  version: 5.4.15
-  resolution: "vite@npm:5.4.15"
+"vite@npm:^5.4.15, vite@npm:^5.4.19":
```

Comment:
Is this correct? I would expect it to not longer reference `5.4.15` anymore.

Reply:
This is because of the transitive workspace dependencies. I don't see a way to automatically update those. I updated them manually.